pages/_app.js

```diff
@@ -1,10 +1,40 @@
 import GlobalStyle from "../styles";
+import useSWR from "swr";
+
+const fetcher = async (url) => {
+  const res = await fetch(url);
+
+  if (!res.ok) {
+    const error = new Error("An error occurred while fetching the data.");
+    error.info = await res.json();
+    error.status = res.status;
+    throw error;
+  }
+
+  return res.json();
+};
 
 export default function App({ Component, pageProps }) {
+  const {
+    data: artworks,
+    error,
+    isLoading,
+  } = useSWR(
+    "https://api.smk.dk/api/v1/art/search/?keys=*&fields=image_thumbnail&fields=titles&fields=id&fields=production&fields=dimensions&fields=current_location_name&fields=production_dates_notes&fields=labels&filters=[image_hq:true],[object_names:painting],[public_domain:true]&offset=0&rows=2000&lang=en",
```

Comment:
Is there a way to limit the number of returned objects from the API? The list is endlessly long, you should think of a way to implement some kind of pagination (probably controlled via offset and rows parameter in the request.

Reply:
Good idea! 😊 Thank you!

Reply:
Wow! I am impressed. Well done, Mona!